IdentityCore/xcconfig/identitycore__idlib__ios.xcconfig

```diff
@@ -1,2 +1,25 @@
 #include "identitycore__lib__common__ios.xcconfig"
 #include "identitycore__idlib__common.xcconfig"
+
+MS_SIM_ARCHS = x86_64 i386
+PRE_XCODE_10_1_ARCHS = armv7 armv7s arm64
+ARCHS[sdk=iphoneos*] = $(PRE_XCODE_10_1_ARCHS) arm64e
+ARCHS[sdk=iphoneos9.*] = $(PRE_XCODE_10_1_ARCHS)
+ARCHS[sdk=iphoneos10.*] = $(PRE_XCODE_10_1_ARCHS)
+ARCHS[sdk=iphoneos11.*] = $(PRE_XCODE_10_1_ARCHS)
+ARCHS[sdk=iphoneos12.0] = $(PRE_XCODE_10_1_ARCHS)
+ARCHS[sdk=iphonesimulator*] = $(MS_SIM_ARCHS)
+
+VALID_ARCHS[sdk=iphoneos*] = $(ARCHS)
+VALID_ARCHS[sdk=iphoneos9.*] = $(ARCHS)
+VALID_ARCHS[sdk=iphoneos10.*] = $(ARCHS)
+VALID_ARCHS[sdk=iphoneos11.*] = $(ARCHS)
+VALID_ARCHS[sdk=iphoneos12.0] = $(ARCHS)
+VALID_ARCHS[sdk=iphonesimulator*] = $(MS_SIM_ARCHS)
+
+// Use empty bitcode marker in simulators and debug configurations for real devices.
+OTHER_CFLAGS[sdk=iphonesimulator*] = $(OTHER_CFLAGS) -fembed-bitcode-marker
```

Comment:
Why is custom configuration here necessary?
I'd prefer to use default Apple configurations as much as possible, since it minimizes a chance of breaking as new xcode gets released.

Reply:
Are you referring to all the bitcode configuration. As a library we need to enable full bitcode for real devices in our release build. This is why it's enforced in the config file.

Also as an optimization bitcode markers only are enough for simulators in any configuration and for devices in debug configuration.

Reply:
Isn't it enabled by default? Just not sure why we need to have custom options here.

Reply:
I removed this one but I have to keep `bitcode` for release since the library should contain full bitcode in the library at the time the library is built.